src/CalendarApp.Domain/Services/MeetingUpdaterService.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using CalendarApp.Contracts;
+using CalendarApp.DataAccess.Repositories.Interfaces;
+using CalendarApp.Domain.Exceptions;
+using CalendarApp.Domain.Services.Interfaces;
+
+namespace CalendarApp.Domain.Services;
+
+public class MeetingUpdaterService
+{
+    public static Meeting FindMeetingByName(string meetingName)
+    {
+        IEnumerable<Meeting> allMeetings = Factory.MeetingsService.GetAllMeetings();
+
+        foreach (Meeting meeting in allMeetings)
```

Comment:
it can be rewritten with single linq statement
```suggestion
        allMeetings.FirstOrDefault(x => x.Name.Equals(meetingName))
```